<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置中心</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="StyleSheetableWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>748</width>
     <height>548</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>748</width>
     <height>548</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>22</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_logo">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_title">
             <property name="text">
              <string>华宇拼音输入法</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="miniBtn">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="close_push_button">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="config_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>基本</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="advanced_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>高级</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="skin_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>外观</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wubi_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>五笔</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wordlib_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>词库</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="account_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="about_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>关于</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="feedback_option_button">
            <property name="minimumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>165</width>
              <height>45</height>
             </size>
            </property>
            <property name="text">
             <string>反馈</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resetconfigbtn">
            <property name="text">
             <string>恢复本页默认设置</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="Line" name="line">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>25</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="bottom_widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>20</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="confirm_button">
               <property name="minimumSize">
                <size>
                 <width>85</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="cancel_button">
               <property name="minimumSize">
                <size>
                 <width>85</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="apply_button">
               <property name="minimumSize">
                <size>
                 <width>85</width>
                 <height>31</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>85</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>应用</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>15</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StyleSheetableWidget</class>
   <extends>QWidget</extends>
   <header>customize_ui/stylesheetablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
